OpenAdaptAI/OpenAdapt
# feat: Add posthog analytics

**What kind of change does this PR introduce?**

This PR introduces analytics to the web dashboard.



**Summary**

It uses [posthog](https://posthog.com/) for tracking user interactions, so we can better understand how users navigate around the dashboard, which should allow us to focus on the important stuff.



**Checklist**

**How can your code be run and tested?**



Sign up on https://posthog.com/ and fetch the public key, paste in in `openadapt/app/dashboard/app/providers.tsx`. Move around in the dashboard, clicking on stuff, changing pages, etc. Look into the posthog dashboard and verify that the events are being captured


**Other information**
